Description: Use non-versionned libusb.h path
 Reference: http://thread.gmane.org/gmane.comp.lib.libusb.devel.general/11611
 .
 Hopefully fixes the kFreeBSD FTBFS.
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2013-03-03
--- a/libfprint/drivers/vfs301_proto.h
+++ b/libfprint/drivers/vfs301_proto.h
@@ -18,7 +18,7 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
-#include <libusb-1.0/libusb.h>
+#include <libusb.h>
 
 enum {
 	VFS301_DEFAULT_WAIT_TIMEOUT = 300,
